src/main.rs

```diff
@@ -80,21 +80,24 @@ fn main() -> Result<()> {
 
     let kernel_version = KernelVersion::current()?;
     let _owned_fd: OwnedFd;
-    let stats_syscall_supported = kernel_version.ge(&KernelVersion::new(5, 8, 0));
+    let mut stats_enabled_via_procfs = false;
 
-    // enable BPF stats via syscall if supported
-    // otherwise, enable via procfs
-    if stats_syscall_supported {
+    // enable BPF stats via syscall if kernel version >= 5.8
+    if kernel_version.ge(&KernelVersion::new(5, 8, 0)) {
```

Comment:
Does KernelVersion implement `Ord` or `PartialOrd` because in that case you can write this as: 

```rust
if kernel_version > KernelVersion::new(5,8,0)
```

Reply:
It [does](https://docs.rs/procfs/latest/procfs/sys/kernel/struct.Version.html#impl-Ord-for-Version)